pkgs/development/python-modules/mathutils/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "mathutils";
+  version = "3.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-uTca+BSfPz8F7hZctYOy33s/vJk1QG0cUFpSdrjlY68=";
+  };
+
+  pythonImportsCheck = [ "mathutils" ];
+
+  meta = with lib; {
+    description = "Utilities for working with datetime objects.";
+    homepage = "https://gitlab.com/ideasman42/blender-mathutils";
+    license = licenses.gpl2;
```

Comment:
gpl2 is deprecated, please use one of gpl2Only, gpl2Classpath, gpl2ClasspathPlus, etc ... see  lib/licenses.nix

Reply:
Changed it to gpl2Plus.